Vinyl/Extensions/Response.swift

```diff
@@ -14,11 +14,11 @@ extension Response {
         guard
             let urlString = encodedResponse["url"] as? String,
             let url =  NSURL(string: urlString),
-            let statusCode = encodedResponse["statusCode"] as? Int,
```

Comment:
Can't we support both? I rather use our approach/naming which I find more meaningful :disappointed: